src/screens/VaultScreen.js

```diff
@@ -1,23 +1,373 @@
+import { Font } from 'expo';
 import Icon from '@expo/vector-icons/FontAwesome';
-import { StyleSheet, Text, View } from 'react-native';
 import { imageStyles } from '../styles/images';
 import { layoutStyles } from '../styles/layout';
 
 export default class VaultScreen extends React.Component {
-
   static navigationOptions = {
     tabBarLabel: 'Vault',
     tabBarIcon: ({ tintColor }) => (
       <Icon name="briefcase" size={22} color="#e91e63" />
     ),
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchText: '',
+      filter: [],
+      categories: [
+        {name: 'CLEAR', selected:false, id: 1},
+        {name: 'FINANCE', selected:true, id: 2},
+        {name: 'HOUSING', selected:true, id: 3},
+        {name: 'EMPLOYMENT', selected:true, id: 4},
+        {name: 'LEGAL', selected:true, id: 5},
+        {name: 'MENTAL HEALTH', selected:true, id: 6},
+      ],
+      fontLoaded: false,
+      resources: [
+        /*
+        {title: 'Title of Resource',
+          partner_org: 'Name of Partner Org',
+          date:'8 Oct 2018',
+          description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. In et aliquam neque. Proin lectus neque, tincidunt eget elementum sit amet, rhoncus ut libero.',
+          link: 'something',
+          upvotes: 0,
+          category: 2,
+          id: 1,
+        },
+        */
+      ]
+    };
+    this.falseState = this.falseState.bind(this);
+    this.setOpposite = this.setOpposite.bind(this);
+    this.renderScrollView = this.renderScrollView.bind(this);
+  }
+
+  async componentDidMount() {
+    await Font.loadAsync({
+      'source-sans-pro-black': require('../.././assets/fonts/SourceSansPro-Black.ttf'),
+      'source-sans-pro-black-italic': require('../.././assets/fonts/SourceSansPro-BlackItalic.ttf'),
+      'source-sans-pro-bold': require('../.././assets/fonts/SourceSansPro-Bold.ttf'),
+      'source-sans-pro-bold-italic': require('../.././assets/fonts/SourceSansPro-BoldItalic.ttf'),
+      'source-sans-pro-extra-light': require('../.././assets/fonts/SourceSansPro-ExtraLight.ttf'),
+      'source-sans-pro-extra-light-italic': require('../.././assets/fonts/SourceSansPro-ExtraLightItalic.ttf'),
+      'source-sans-pro-italic': require('../.././assets/fonts/SourceSansPro-Italic.ttf'),
+      'source-sans-pro-light': require('../.././assets/fonts/SourceSansPro-Light.ttf'),
+      'source-sans-pro-light-italic': require('../.././assets/fonts/SourceSansPro-LightItalic.ttf'),
+      'source-sans-pro-regular': require('../.././assets/fonts/SourceSansPro-Regular.ttf'),
+      'source-sans-pro-semibold': require('../.././assets/fonts/SourceSansPro-SemiBold.ttf'),
+      'source-sans-pro-semibold-italic': require('../.././assets/fonts/SourceSansPro-SemiBoldItalic.ttf'),
+    });
+    this.setState({fontLoaded: true});
+  }
+
+  updateFilter(itemId, newState) {
+    /**
```

Comment:
ooh comments look great, can you put all the comments above the function name, not inside the function?